fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/provider/impl/S3Provider.java

```diff
@@ -186,11 +203,23 @@ public ListObjectsV2Result getListObject(String prefix, String continuationToken
 
     @Override
     public long getSize(String workItem) throws FHIRException {
+        return getSize(client, bucketName, workItem);
+    }
+
+    /**
+     * Get Size enables multiple use case beyond the default of getSize
+     * @param c
+     * @param bn
+     * @param workItem
+     * @return
+     * @throws FHIRException
+     */
+    public long getSize(AmazonS3 c, String bn, String workItem) throws FHIRException {
```

Comment:
What units for size? Bytes or characters?

Reply:
bytes - added implNote